bucket/monero-cli.json

```diff
@@ -0,0 +1,55 @@
+{
+    "homepage": "https://getmonero.org/",
+    "description": "The secure, private, untraceable cryptocurrency (CLI)",
+    "version": "0.14.1.0",
+    "license": "BSD-3-Clause",
+    "architecture": {
+        "64bit": {
+            "url": "https://dlsrc.getmonero.org/cli/monero-win-x64-v0.14.1.0.zip",
+            "hash": "ca4367c565026ea23bf90180cdd750a1d6adfab36ee19435e91ed740a369a6fc",
+            "extract_dir": "monero-x86_64-w64-mingw32"
+        },
+        "32bit": {
+            "url": "https://dlsrc.getmonero.org/cli/monero-win-x86-v0.14.1.0.zip",
+            "hash": "ed26ad4c7f3fc15893c615e438a76f91c8c5c18e29d32bf350e126beecc2877f",
+            "extract_dir": "monero-i686-w64-mingw32"
+        }
+    },
+    "bin": [
+        "monerod.exe",
+        "monero-wallet-rpc.exe",
+        "monero-wallet-cli.exe",
+        "monero-gen-trusted-multisig.exe",
+        "monero-blockchain-usage.exe",
+        "monero-blockchain-mark-spent-outputs.exe",
+        "monero-blockchain-import.exe",
+        "monero-blockchain-export.exe",
+        "monero-blockchain-depth.exe",
+        "monero-blockchain-ancestry.exe",
+        "monero-blockchain-prune.exe",
+        "monero-blockchain-prune-known-spent-data.exe",
+        "monero-blockchain-stats.exe",
+        [
+            "monero-blockchain-mark-spent-outputs.exe",
+            "monero-blockchain-blackball"
+        ]
+    ],
+    "checkver": {
+        "url": "https://getmonero.org/downloads/hashes.txt",
+        "re": "monero-win-x64-v([\\d.]+).zip"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://dlsrc.getmonero.org/cli/monero-win-x64-v$version.zip"
+            },
+            "32bit": {
+                "url": "https://dlsrc.getmonero.org/cli/monero-win-x86-v$version.zip"
+            }
+        },
+        "hash": {
+            "url": "https://getmonero.org/downloads/hashes.txt",
+            "find": "$basename.*([a-fA-F0-9]{64})"
```

Comment:
```suggestion
            "regex": "$basename,\\s+$sha256"
```

Reply:
Why `,\\s+`?

Reply:
`monero-win-x64-v0.14.1.0.zip, ca4367c565026ea23bf90180cdd750a1d6adfab36ee19435e91ed740a369a6fc`

Reply:
`.*` works, too. But okay, let's use exactly match

Reply:
That is nonsense to use greedy `.*` for two characters